drivers/hmis/spec/models/hmis/form/form_processor_spec.rb

```diff
@@ -409,6 +409,34 @@
       expect(disabilities.find_by(disability_type: 8).t_cell_count_available).to eq(1)
     end
 
+    it 'processes HOPWA fields (regression #6864)' do
```

Comment:
Issue to address the fact that this was a silent failure: https://github.com/open-path/Green-River/issues/6868